trie/inspect_trie.go

```diff
@@ -0,0 +1,297 @@
+package trie
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"math/big"
+	"os"
+	"runtime"
+	"sort"
+	"strconv"
+	"sync"
+	"sync/atomic"
+
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	"github.com/ethereum/go-ethereum/core/types"
+	"golang.org/x/sync/semaphore"
+)
+
+const (
+	DEFAULT_TRIEDBCACHE_SIZE = 1024 * 1024 * 1024
```

Comment:
Please don't uppercase constants. It's not go-idiomatic. `DefaultTriedbCacheSize` is better, but also add documentation for things like this

Reply:
And make it non-public unless you have some specific need for it to be public. (So `defaultTriedbCacheSize`)